yaml-schema.json

```diff
@@ -0,0 +1,138 @@
+                                "description": "Default value for the input channel"
+                            }
+                        },
+                        "required": ["type", "description"]
+                    }
+                }
+            }
+        },
+        "output": {
+            "type": "array",
+            "description": "Output channels for the module",
+            "items": {
+                "type": "object",
+                "patternProperties": {
+                    ".*": {
+                        "type": "object",
+                        "properties": {
+                            "type": {
+                                "type": "string",
+                                "description": "Type of the output channel"
+                            },
+                            "description": {
+                                "type": "string",
+                                "description": "Description of the output channel"
+                            },
+                            "pattern": {
+                                "type": "string",
+                                "description": "Pattern of the output channel"
+                            }
+                        },
+                        "required": ["type", "description"]
+                    }
+                }
+            }
+        },
+        "tools": {
+            "type": "array",
+            "description": "Tools used by the module",
+            "items": {
+                "type": "object",
+                "patternProperties": {
+                    ".*": {
+                        "type": "object",
+                        "properties": {
+                            "description": {
+                                "type": "string",
+                                "description": "Description of the output channel"
+                            },
+                            "homepage": {
+                                "type": "string",
+                                "description": "Homepage of the tool"
+                            },
+                            "documentation": {
+                                "type": "string",
+                                "description": "Documentation of the tool"
+                            },
+                            "tool_dev_url": {
+                                "type": "string",
+                                "description": "URL of the development version of the tool's documentation"
+                            },
+                            "doi": {
+                                "type": "string",
+                                "description": "DOI of the tool"
+                            },
+                            "licence": {
+                                "type": ["array", "string"],
+                                "description": "Licence of the tool",
```

Comment:
Maybe an enum using: http://bio-tools.github.io/biotoolsSchema/#Link21 ?

Reply:
Hmm, that list is a bit too long and would make the schema quite unwieldy I think.

Reply:
I thought with JSON schema you can refer to 'remote' enums?   But agree wouldn't want to embed the whole list.

Reply:
I wonder how bioconda/conda does it :thinking:

Reply:
ah, true! will try that